ScnSoft-Education/LeetCodeTraining.Tests/PalindromeNumberSolitionTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using LeetCodeTraining.PalindromeNumber9;
+
+namespace LeetCodeTraining.Tests
+{
+    public class PalindromeNumberSolitionTests
+    {
+        [Fact]
+        public void PalindromeNumber_StringWith3Nums()
```

Comment:
Please use Theory and (InlineData or MemberData) attributes to simplify tests,
as a result  there can be one method with 4 attributes.
Act's part will be moved to attributes.
https://andrewlock.net/creating-parameterised-tests-in-xunit-with-inlinedata-classdata-and-memberdata/